src/main/scala/viper/silver/verifier/ModelParser.scala

```diff
@@ -1,18 +1,24 @@
 package viper.silver.verifier
 
 import fastparse._
-import viper.silver.parser.FastParserCompanion.whitespace
-
 object ModelParser {
+  def modelEntry[_: P]: P[(String, ModelEntry)] = {
+    implicit val ws = viper.silver.parser.FastParserCompanion.whitespaceWithoutNewlineOrComments
```

Comment:
The point of the entire PR is that we need to be able to parse names without values, so
```
name -> 
```
Here, we need the ~ combinator between "->" and definition to accept whitespace inbetween, but no newlines.
Otherwise we could parse the name of the next variable in the next line as the value.
The ~ combinator takes an implicit argument that defines how to parse whitespace; the implicit val will be used here to parse whitespace except for newlines.
That's also why I moved the whitespace import a few lines further down, so that we only have one implicit whitespace definition in scope here.
I'll add a comment to make this explicit.

Reply:
Thanks!